apps/api/src/modules/user/profile/profile.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { EditActorProfileDto, JwtDto, UserType } from '@modela/dtos'
+import { Body, Controller, Put } from '@nestjs/common'
+import {
+  ApiForbiddenResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger'
+import { UseTypeAuthGuard } from 'src/modules/auth/misc/jwt.decorator'
+import { User } from 'src/modules/auth/misc/user.decorator'
+
+import { ProfileService } from './profile.service'
+
+@ApiTags('profile')
+@Controller('profile')
+export class ProfileController {
+  constructor(private readonly profileService: ProfileService) {}
+
+  @Put('/actor')
+  @UseTypeAuthGuard(UserType.ACTOR)
```

Comment:
Note: @UseTypeAuthGuard will can be only use by verified user. I will create new Guard and edit this for the future.